static/js/views/settings.js

```diff
@@ -108,8 +113,57 @@ const SettingsScreen = {
     });
 
     this.setupNetworkElements();
+    this.setupLocalizationElements();
   },
 
+  /* eslint-disable max-len */
+  setupLocalizationElements: function() {
+    this.elements.localization = {
+      country: document.getElementById('localization-settings-country'),
+      timezone: document.getElementById('localization-settings-timezone'),
+      language: document.getElementById('localization-settings-language'),
+      units: {
+        temperature: document.getElementById('localization-settings-units-temperature'),
+      },
+    };
+
+    this.elements.localization.country.addEventListener(
+      'change',
+      (ev) => {
+        API.setCountry(ev.target.value)
+          .catch(console.error);
```

Comment:
Something we could do a better job of in general is providing user feedback (e.g. via a toast) when an operation like this fails. Not necessarily to address in this patch, but the kind of thing I should really be including in a more full UX specification.